src/blade/cc_targets.py

```diff
@@ -907,6 +907,12 @@ def _rpath_link(self, dynamic):
             return os.path.dirname(path)
         return None
 
+    def _need_generate_hdrs(self):
+        for path in self.blade.get_sources_keyword_list():
```

Comment:
get_sources_keyword_list()
这个是什么前几天看过我现在又想不起来了，应该改名

Reply:
目前就是指的第三方目录：
def get_sources_keyword_list(self):
        """This keywords list is used to check the source files path.

        Ex, when users specifies warning=no, it could be used to check that
        the source files is under thirdparty or not. If not, it will warn
        users that this flag is used incorrectly.

        """
        keywords = ['thirdparty']
        return keywords